complete/src/main/java/com/example/restservice/service/TokenService.java

```diff
@@ -81,8 +81,12 @@ public TokenDetailResponse createToken(CreateTokenRequest createTokenRequest) {
                   return newToken;
                 })
             .orElseThrow(SQInvalidRequestException::queueNotFoundException);
+    Integer currentMaxTokenNumber = tokenRepository.getLastTokenNumberForQueue(token.getQueue().getQueueId());
+    int nextTokenNumber = currentMaxTokenNumber != null ? currentMaxTokenNumber + 1 : 1;
+    token.setTokenNumber(nextTokenNumber);
```

Comment:
These are nettigritty (nit for short) conventions that we follow... Always use `var`. And usually if a value is used only once, directly use it, unless there is a strong readability advantage. So the above lines will become:
```
var currentMaxTokenNumber = tokenRepository.getLastTokenNumberForQueue(token.getQueue().getQueueId());
token.setTokenNumber(currentMaxTokenNumber != null ? currentMaxTokenNumber + 1 : 1);
```